tests/contrib/aiohttp/test_aiohttp_patch_generated.py

```diff
@@ -19,6 +19,7 @@ class TestAiohttpPatch(PatchTestCase.Base):
     __module_name__ = "aiohttp"
```

Comment:
We can do this in a follow up but the file name should be `tests/contrib/aiohttp/test_aiohttp_patch.py` (remove the _generated.py)